railties/test/generators/app_generator_test.rb

```diff
@@ -940,6 +940,23 @@ def test_skip_dev_gems
     assert_no_gem "web-console"
   end
 
+  def test_no_skip_bcrypt_with_minimal
+    run_generator [destination_root, "--minimal", "--no-skip-bcrypt"]
+    assert_gem "bcrypt"
+  end
+
+  def test_skip_bcrypt_default
+    run_generator [destination_root]
```

Comment:
```suggestion
    run_generator
```

`[destination_root]` is the default argument passed to `run_generator`